Reservia_files/style.css

```
@@ -0,0 +1,620 @@
+    font-family: 'Raleway', sans-serif;
+}
+
+
+
+body{ }
+
+a{  font-size: 16px; font-weight: bold; text-decoration: none; }
+
+
+
+
+
+/** --------------- HEADER --------------- **/
+
+.pageHead{
+    max-width: 1440px;
+    width: 100%;
+    padding: 0px 50px;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-end;
+    align-content: flex-end;
+    height: 85px;
+    margin: auto;
+    margin-bottom: 50px;
+}
+
+.pageHead__logo{
+    width: 94px;
+    margin-right: auto;
+	
+}
+
+.pageHead__navItem{
+    padding: 0px 35px;
+    font-weight: normal;
+    Color: black;
+	
+}
+
+.pageHead__navItem:hover{
+    color: #0065FC;
+	/*color: yellow;*/
+    border-top: 2px solid #0065FC;
+    padding-top: 39px;
+}
+
+.pageHead__itemRegister{
+    color: #0065FC ;
+    padding: 0px 39px;
+	padding-right: 90px;
+}
+/** --------------- NAVIGATION --------------- **/
+
+
+#garden {
+    display: grid;
+    grid-template-columns: 3% 50% 30% ;
+    /*grid-template-rows: 20% 20% 20% 20% 20%;*/
+  }
+
+  .fit-picture{
+    grid-column-start: 1;
+  }
+  #name{
+    grid-column-start: 2;
+  }
+  #ida{
+    grid-column-start: 3;
+  }
+.fit-picture{
+    border-top-left-radius: 30%;
+    border-bottom-left-radius:30%;
+    height:35px;
+}
+input{
+    width:96%;
+    height:29px;
+    border-color:#0065FC;
+}
+#PageRelease{
+padding-left: 3%;
+}
+.button>button{
+background-color:#0065FC;
+width:20%;
+    height:35px;
+    border-top-right-radius: 12%;
+    border-bottom-right-radius:12%;
+    border:none;
+    grid-column-start: 3;
+
+}
+/*.fit-picture{
+    border-radius: 45%;
+    width:50px;
+    height:50px;
+}*/
+
+.Informationpicture{
+    width: 30px;
+	/*height: 50px;*/
+    
+}
+.buttonFilter>button{
+    width: 400px; height: 50px;
+  
+}
+.pictureButton{
+    
+    border:none;
+    background:white;
+    
+}
+ .pictureButton>#img1{
+     width: 140px;
+ }
+ .pictureButton>#img2{
+    width: 120px;
+   
+}
+.pictureButton>#img3{
+    width: 140px;
+}
+.pictureButton>#img4{
+    width: 180px;
+}
+
+span{
+    font-weight: bold;
+    
+    
+}
+.label{
+    float:left;
+    
+}
+
+/* ------Host------*/
+.section1{
+    display: grid;
+    background-color:white;
+    
+    grid-template-columns: 60% 40%  ;
+    /*grid-template-rows: 20% 20% 20% 20% 20%;*/
+    
+}
+
+.section1>#box1{
+    grid-column-start: 1;
+    background-color:#F2F2F2;
+    border-radius: 2%;
+    display:grid;
+    grid-template-columns: 33,33% 33,33% 33,33% ;
+    grid-template-rows: 50% 50% ;
+    padding-left: 4%;
+    padding-right: 4%;
+    padding-top:8%;
+    padding-bottom: 8%;
+    margin-left: 5%;  
+}
+
+.section1>#box2{
+    
+    grid-column-start: 2;
+    background-color:#F2F2F2;
+    margin-left: 7%;
+    padding-left: 4%;
+    padding-right: 4%;
+    padding-top:15%;
+    padding-bottom: 4%;
+    border-radius: 2%;
+    display:grid;
+    grid-template-rows: 30% 30% 30% ;
+    margin-right: 5%;
+}
+.section1>#box1>#sousbox1-1{
```

the selectors bellow
are using almost the same css properties! you should try to create a class that include margins-tops etc... =2% and background color white and finally use ids only for grid-column position.

without even changing your html you could do following for applying the margins and background color.
```suggestion
#box1>[id^="sousbox"]{
    background-color:white;    
    grid-column: 1;
    border-radius: 2%;
    margin-top: 2%;    
    margin-left: 2%;    
    margin-right: 2%;    
    margin-bottom: 2%;
}
```
Some more details about this kind of selectors  ( it's a must read) https://developer.mozilla.org/fr/docs/Web/CSS/Attribute_selectors


and then in your #sousbox1-1 you can just choose the column number but i would advise you to get ride telling which div is which column and to use automatic placement https://developer.mozilla.org/fr/docs/Web/CSS/CSS_Grid_Layout/Auto-placement_in_CSS_Grid_Layout

---

I proceeded to the factorization of the code, according to the repetition of the code.